<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clemap Sensor Demo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>clemap_icon.png</normaloff>clemap_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBox_l1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>L1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_l2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>L2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_l3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>L3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_p">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wirkleistung</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_q">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Blindleistung</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_i">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Strom</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_v">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Spannung</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_phas">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phasen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_para">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameter</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>100</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_out">
    <property name="geometry">
     <rect>
      <x>926</x>
      <y>710</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Extreme Outliers Off</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_axis">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Label</string>
    </property>
   </widget>
   <widget class="QLabel" name="x_axis">
    <property name="geometry">
     <rect>
      <x>966</x>
      <y>670</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zeit (hh:mm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>258</x>
      <y>28</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string> 3-Phasen-Energiemonitoring-Sensor Demo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>37</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>clemap_logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_min">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>553</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_max">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>593</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_avg">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>633</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Avg:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_med">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>673</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Med:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>520</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data </string>
    </property>
   </widget>
   <widget class="QLabel" name="min_l1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>553</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="max_l1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>593</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="avg_l1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>633</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="med_l1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>673</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="min_l2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>553</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="max_l2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>593</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="avg_l2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>633</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="med_l2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>673</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="med_l3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>673</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="max_l3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>593</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="avg_l3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>633</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="min_l3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>553</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>L1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>L2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>520</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>L3</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>509</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <zorder>graphicsView_2</zorder>
   <zorder>checkBox_l1</zorder>
   <zorder>checkBox_l2</zorder>
   <zorder>checkBox_l3</zorder>
   <zorder>checkBox_p</zorder>
   <zorder>checkBox_q</zorder>
   <zorder>checkBox_i</zorder>
   <zorder>checkBox_v</zorder>
   <zorder>label_phas</zorder>
   <zorder>label_para</zorder>
   <zorder>graphicsView</zorder>
   <zorder>checkBox_out</zorder>
   <zorder>y_axis</zorder>
   <zorder>x_axis</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_min</zorder>
   <zorder>label_max</zorder>
   <zorder>label_avg</zorder>
   <zorder>label_med</zorder>
   <zorder>label_7</zorder>
   <zorder>min_l1</zorder>
   <zorder>max_l1</zorder>
   <zorder>avg_l1</zorder>
   <zorder>med_l1</zorder>
   <zorder>min_l2</zorder>
   <zorder>max_l2</zorder>
   <zorder>avg_l2</zorder>
   <zorder>med_l2</zorder>
   <zorder>med_l3</zorder>
   <zorder>max_l3</zorder>
   <zorder>avg_l3</zorder>
   <zorder>min_l3</zorder>
   <zorder>label_16</zorder>
   <zorder>label_17</zorder>
   <zorder>label_18</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
